alibi/explainers/ale.py

```diff
@@ -112,12 +111,11 @@ def explain(self,
             Minimum number of points each discretized interval should contain to ensure more precise
             ALE estimation.
         grid_points
```

Comment:
I just realized that perhaps we should add to the various parameters in the constructor docstring controlling the quantile selection behaviour that they are only relevant if `grid_points` is `None` (well it's actually relevant feature-wise...)? Or perhaps a better way to add to the `grid_points` docstring that for the features for which `grid_points` are passed the relevant parameters passed in the constructor are not considered?